projects/rocprofiler-systems/source/lib/core/rocpd/data_storage/database.cpp

```diff
@@ -93,9 +93,7 @@ database::initialize_schema()
                 return new_file_path;
             }
         }
-        return std::string(
-                   "rocprofiler-systems/source/lib/core/rocpd/data_storage/schema/")
-            .append(filename);
+        return std::string("source/lib/core/rocpd/data_storage/schema/").append(filename);
```

Comment:
```suggestion
        // TODO:  Update to look for the system's rocpd schema
        return std::string("source/lib/core/rocpd/data_storage/schema/").append(filename);
```
This is fine for bring-up or a debug build, but eventually we'll need to update this to look for the schema installed by the system's "rocpd" version.